xrayvision/tests/test_imaging.py

```diff
@@ -93,7 +93,7 @@ def test_vis_to_psf(pixel_size, uv):
     weights = np.sqrt(u**2 + v**2).value
     weights /= weights.sum()
     psf_calc = idft_map(obs_vis, u=u, v=v, shape=[65, 65] * apu.pix, pixel_size=ps, weights=weights)
-    vis = Visibility(obs_vis, u=u, v=v)
+    vis = Visibilities(obs_vis, u, v)
```

Comment:
Since u and v are not optional parameters, I thought it was necessary to do this. Does it work also as `Visibilities(obs_vis, u=u, v=v)`? I haven't tried

Reply:
Yea we should probably make them keyword only